src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts

```diff
@@ -233,22 +213,18 @@ registerActiveXtermAction({
 registerActiveXtermAction({
 	id: TerminalChatCommandId.ViewInChat,
 	title: localize2('viewInChat', 'View in Chat'),
+	category: AbstractInlineChatAction.category,
 	precondition: ContextKeyExpr.and(
 		ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
 		TerminalChatContextKeys.requestActive.negate(),
 	),
 	icon: Codicon.commentDiscussion,
 	menu: [{
 		id: MENU_TERMINAL_CHAT_WIDGET_STATUS,
-		group: '0_main',
+		group: 'zzz',
 		order: 1,
+		isHiddenByDefault: true,
```

Comment:
should other actions, like `insertFirstCommand` and `run` also be hidden by default?